content/en/docs/reference/config/analysis/ist0001/index.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
Is this file supposed to be here? All the other errors don't have a corresponding index.md file, so I assume their contents is generated from analysis.yaml.

Reply:
This is a placeholder to show how to add messages.

We'll need to add one .md file per message. This is work @sushicw and co will take on. I'm just laying the structure.